torch/quantization/observer.py

```diff
@@ -532,8 +538,12 @@ def forward(self, x_orig):
             min_val, max_val = torch._aminmax(x)
         else:
             min_val_cur, max_val_cur = torch._aminmax(x)
-            min_val = min_val + self.averaging_constant * (min_val_cur - min_val)
-            max_val = max_val + self.averaging_constant * (max_val_cur - max_val)
+            min_val = min_val.to(device=min_val_cur.device) + \
```

Comment:
Can the code to move `min_val`/`max_val` be moved outside this statement so we do it only once?